src/components/misc/FieldSelector.tsx

```diff
@@ -11,7 +11,7 @@ type Props = {
   onChange: (label: string | undefined) => void;
```

Comment:
Misc also contains components, so it should be capitalized. I'd maybe consider UiElements or put them inside `components`, which should also be capitalized.

Reply:
renaming `misc` to `UiElements`? Fine with that.

Reply:
I don't like it either honestly. Maybe I would let them just be under Components, or create a few folders that are more specific (misc is ~= utils).

For example, form or form controls:
- AddToFiltersButton
- FieldSelector
- GoToExploreButton
- SelectFieldButton
- ShareExplorationButton
- LiveTailCONTROL
- LayoutSwitcher
- LineFilter
- GiveFeedback (it's a button)

Ui/UIState/Messages
- GrotError
- StatusWrapper
-